minutes/2021-01-12.md

```diff
@@ -0,0 +1,50 @@
+# Jakarta EE Platform Call
+
+Date: 2021-01-12
+
+Present:
+
+- Ivar Grimstad (Eclipse Foundation)
+- Andy McCright (IBM)
+- Kevin Sutter (IBM)
+- Nitin Dahyabhai (Eclipse WTP)
+- Brian Stansberry (Red Hat)
+- David Blevins (Tomitribe)
+- Jean-Louis Monteiro (Tomitribe)
+- Nathan Erwin (Individual)
+- Dmitry Kornilov (Oracle)
+- Scott Marlow (Red Hat)
+- Cesar Hernandez (Tomitribe)
+- Ed Bratt (Oracle)
+- Kenji Kazumura (Fujitsu)
+- Steve Millidge (Payara)
+- Thomas Watson (IBM)
+- Scott Stark (Red Hat)
+- Ryan Cuprak (Jakarta EE Ambassadors)
+- Tanja Obradovic (Eclipse Foundation)
+- Werner Keil (Individual)
+
+## Agenda and Minutes
+
+### [IG] Individual Specifications Plan Reviews
+
+* Deadline for potential inclusion in 9.1: January 31
+*Ä None so far
```

Comment:
An errant character here after the asterisk?

Reply:
Thanks :)